core/scripts/pptr-run-devtools.js

```diff
@@ -300,12 +300,17 @@ function dismissDialog(dialog) {
  * @return {Promise<{lhr: LH.Result, artifacts: LH.Artifacts, logs: string[]}>}
  */
 async function testUrlFromDevtools(url, options = {}) {
-  const {config, chromeFlags} = options;
+  const {config, chromeFlags = []} = options;
 
-  const browser = await puppeteer.launch({
```

Comment:
Puppeteer will now disable the bfcache if we aren't using tab target which is currently only available through an environment variable `PUPPETEER_INTERNAL_TAB_TARGET`. The env variable needs to be set *before puppeteer is imported* so we would need to do some annoying dynamic imports.

This is an alternative approach which just uses `chrome-launcher` with our desired flags. This should be better for our smoke tests in general because it's closer to how Lighthouse launches Chrome.